CONTRACT.md

```diff
@@ -69,13 +127,14 @@ Cumulus Messages come in 2 flavors: The full **Cumulus Message** and the **Cumul
 
 #### Cumulus Remote Message example:
 
-The message may contain a reference to an S3 Bucket and Key, as follows:
+The message may contain a reference to an S3 Bucket, Key and TargetPath as follows:
 
 ```json
 {
   "replace": {
     "Bucket": "cumulus-bucket",
     "Key": "my-large-event.json"
```

Comment:
Need to add a comment to the end of this line, which is why Github is flagging the next line as invalid (highlighted in red)